src/services/fileStorage/strategies/awsS3.js

```diff
@@ -528,6 +528,41 @@ class AWSS3Strategy extends AbstractFileStorageStrategy {
 				return Promise.resolve(deletionData);
 			});
 	}
+
+	async moveFilesToTrash(schoolId, fileIds) {
```

Comment:
where is this method used?

Reply:
not at all so far, its preparation for a seperate ticket